### DISCUSSION TITLE
Initial Install - Now How to Access

### DISCUSSION THREAD

**Discussion by imager2019** [inventree/InvenTree#5198] [Q&A] [answered]
I have just set up InvenTree on a VM instance on Debian 11 - using the Fast Install method, and everything seems to have gone well.

On another system, I am able to hit the InvenTree Sign In page, but... I cannot find details on default username / password or if I need to set these up separately.

Is there a default setting initially configured that I missed what the credentials were, or is there a separate process I need to follow to add accounts? Maybe some documents somewhere that I missed? 

John

**Comment by SchrodingersGat** [comment, marked as answer]
https://docs.inventree.org/en/latest/start/install/?h=superuser#create-admin-account

**Reply by imager2019** [reply]
Excellent - thank you. I was able to set a new admin account and get things going.
